lazy_loader/__init__.py

```diff
@@ -99,24 +98,25 @@ def __dir__():
 
 
 class DelayedImportErrorModule(types.ModuleType):
-    def __init__(self, frame_data, *args, **kwargs):
+    def __init__(self, frame_data, *args, message, **kwargs):
         self.__frame_data = frame_data
+        self.__message = message
```

Comment:
> Do you want a default message?

I found it was easier to be more explicit about setting the message in the branching logic. I can rework with a default, if preferred.

> Also, why is message after *args in the function sig instead of before? For backward compat?

I figured keyword-only would be clearer, but I'm okay with any signature. Let me know if you'd like me to change this.

Reply:
Yes, I see what you mean and agree that dealing with the message is better handled in the branching logic.